modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/util/test/JournalConverterImplTest.java

```diff
@@ -403,6 +405,43 @@ public void testGetFieldsFromEmptyContent() throws Exception {
 			_journalConverter.getDDMFields(_ddmStructure, StringPool.BLANK));
 	}
 
+	@Test
+	public void testGetFieldsFromRearrangedStructure() throws Exception {
```

Comment:
Brian might like the method name `testGetDDMFieldsFromRearrangedDDMStructure` better. `GetDDMFields` is the method-under-test and `DDMStructure` is the className for the UI term Structure